test/util/connectors/ConnectorSpec.scala

```diff
@@ -20,14 +20,12 @@ import base.Injector
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.time.{Millis, Seconds, Span}
 import org.scalatest.{BeforeAndAfterEach, MustMatchers, WordSpec}
-import org.scalatestplus.mockito.MockitoSugar
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.bootstrap.http.DefaultHttpClient
 
 import scala.concurrent.ExecutionContext
 
-class ConnectorSpec
-    extends WordSpec with WiremockTestServer with Injector with MustMatchers with MockitoSugar with BeforeAndAfterEach with ScalaFutures {
+class ConnectorSpec extends WordSpec with WiremockTestServer with Injector with MustMatchers with BeforeAndAfterEach with ScalaFutures {
```

Comment:
Nit: Don't think it requires separate `connector` package if it is already moved to `util`.